clang/test/CodeGen/Mips/unreachable.cpp

```diff
@@ -0,0 +1,13 @@
+// REQUIRES: mips-registered-target
```

Comment:
https://maskray.me/blog/2021-08-08-toolchain-testing#the-test-checks-too-little#the-test-checks-at-the-wrong-layer

Should generally avoid clang/test/CodeGen test for lib/Target changes. llvm/test/MC/Mips testing is in a pretty bad state. You might want to check RISCV/LoongArch.

(I will have limited internet access between April 20th and May 4th, and my response time may be delayed..)

Reply:
`clang++ -target "mipsel-w64-windows-gnu" -c test.cpp` avoid deprecated `-target `. Use `--target=`